.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,44 @@
+#
+# This agent will automatically record (some) CTO.ai Delivery & LifeCycle
+# Metrics for use with the CTO.ai Delivery Metrics dashboard.
+#
+# Installation:
+#
+#   1. GitHub -> Settings -> Secrets -> New Secret
+#     - Create CTOAI_TEAM_ID secret using your CTO.ai-issued Team Id.
+#     - Create CTOAI_EVENTS_API_TOKEN secret using your CTO.ai-issued API Token.
```

Comment:
do we still have the path for users to generate long-lived token? Do we still want to go down this path?

Reply:
Yes, they generate via the CLI.